.github/workflows/coverage.yml

```diff
@@ -46,7 +47,7 @@ jobs:
             python3.10-dbg
       - name: Install Python dependencies
         run: |
-          python3 -m pip install --upgrade pip cython
+          python3 -m pip install --upgrade pip cython pkgconfig
```

Comment:
Nope, I'm wrong, we do need it. Not for the `make test-install`, but because later we do `make ccoverage`, which does:
```cpp
CFLAGS=" -O0 -pg --coverage" make build
```
and that needs it.

Sorry for the bad advice!